seekret-rules/mandrill.rule

```diff
@@ -2,7 +2,7 @@ mandrill_username:
   match: (\"|')?(MANDRILL|mandrill)_?(\"|')?(USERNAME|username)?(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9_\-\.]+\@[A-Za-z0-9_\-\.]+(\"|')?
 
 mandrill_password:
-  match: (\"|')?(MANDRILL|mandrill)?_?(\"|')?(PASSWORD|password)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9\+\!\^\%\*\-_]{21,}(\"|')?
+  match: (\"|')?(MANDRILL|mandrill)_?(\"|')?(PASSWORD|password)(\"|')?\s*(:|=>|=)\s*(\"|')?[A-Za-z0-9\+\!\^\%\*\-_]{21,}(\"|')?
```

Comment:
This should probably match passwords that start at 8 characters. Some examples we've found on Github are all about 16 characters. Also, because all the documentation seems to point to `MANDRILL_xyz` as the name of the variable. We should probably not make the `MANDRILL_*` part optional.